spring-cloud-deployer-resource-support/src/main/java/org/springframework/cloud/deployer/resource/support/DelegatingResourceLoader.java

```diff
@@ -117,7 +118,9 @@ public Resource getResource(String location) {
 				return resource;
 			}
 			else {
-				String cacheName = scheme + "-" + ShaUtils.sha1(location) + "-" + resource.getFilename();
+				String fileName = resource.getFilename();
```

Comment:
Yeah, good idea. Given we display the cache file name in the console, I think this is good.